packages/typescript-estree/src/create-program/createProjectService.ts

```diff
@@ -65,6 +65,18 @@ export function createProjectService(
     setTimeout,
     watchDirectory: createStubFileWatcher,
     watchFile: createStubFileWatcher,
+
+    // We stop loading any TypeScript plugins by default, to prevent them from attaching disk watchers
+    // See https://github.com/typescript-eslint/typescript-eslint/issues/9905
+    ...(!options.loadTypeScriptPlugins && {
```

Comment:
Will this not just spread in `false`?

Reply:
`{ ...false }` just becomes `{}`. I've started using this little trick in a lot of spreads, heh.